internal/config/backup_test.go

```diff
@@ -48,39 +48,40 @@ func TestBackupManager_Bind(t *testing.T) {
 		return nil
 	}
 	tests := []test{
-		// TODO test cases
-		/*
-		   {
-		       name: "test_case_1",
-		       fields: fields {
-		           Host: "",
-		           Port: 0,
-		           Client: GRPCClient{},
-		       },
-		       want: want{},
-		       checkFunc: defaultCheckFunc,
-		   },
-		*/
-
-		// TODO test cases
-		/*
-		   func() test {
-		       return test {
-		           name: "test_case_2",
-		           fields: fields {
-		           Host: "",
-		           Port: 0,
-		           Client: GRPCClient{},
-		           },
-		           want: want{},
-		           checkFunc: defaultCheckFunc,
-		       }
-		   }(),
-		*/
+		{
+			name: "return Backup when client is nil",
+			fields: fields{
+				Client: nil,
+			},
+			want: want{
+				want: &BackupManager{
+					Client: newGRPCClientConfig(),
+				},
+			},
+		},
+		func() test {
+			c := &GRPCClient{
+				HealthCheckDuration: "1s",
+			}
+			return test{
+				name: "return Backup when client is not nil",
+				fields: fields{
+					Client: c,
+				},
+				want: want{
+					want: &BackupManager{
+						Client: c.Bind(),
+					},
+				},
+			}
```

Comment:
Could you please add test that the data is loaded from environment variable ?:thinking: